--- mcelog-1.0pre3/paths.h.orig	2010-01-21 03:36:52.000000000 +0100
+++ mcelog-1.0pre3/paths.h	2010-02-01 22:38:22.000000000 +0100
@@ -1,7 +1,7 @@
 #define PREFIX ""
 
 #define LOG_DEV_FILENAME "/dev/mcelog"
-#define DIMM_DB_FILENAME PREFIX "/var/lib/memory-errors"
+#define DIMM_DB_FILENAME PREFIX "/var/lib/misc/memory-errors"
 #define CONFIG_FILENAME PREFIX "/etc/mcelog/mcelog.conf"
 
 #define SOCKET_PATH "/var/run/mcelog-client"
