--- mcelog-0.8pre/Makefile~	2008-10-08 19:13:28.000000000 +0300
+++ mcelog-0.8pre/Makefile	2008-10-08 19:14:11.788165768 +0300
@@ -16,8 +16,8 @@
 
 # dbquery intentionally not installed by default
 install: mcelog
-	cp mcelog ${prefix}/sbin/mcelog
-	cp mcelog.8 ${prefix}/share/man/man8
+	install mcelog $(DESTDIR)${prefix}/sbin/mcelog
+	cp mcelog.8 $(DESTDIR)${prefix}/share/man/man8
 ifdef DOCDIR
 	cp ${DOC} ${DOCDIR} 
 else
